EPPlus/ExcelTheme.cs

```diff
@@ -1,81 +1,92 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
 using System.Xml;
 
 namespace OfficeOpenXml
 {
     public sealed class ExcelTheme : XmlHelper
     {
+        private const string c_node_accent1 = "a:accent1";
```

Comment:
Why not have this as a static array.  This would really clean up the rest of the code